pkg/collector/corechecks/net/network/network.go

```diff
@@ -313,8 +731,12 @@ func Factory() option.Option[func() check.Check] {
 }
 
 func newCheck() check.Check {
+	procfsPath := "/proc"
+	if pkgconfigsetup.Datadog().IsConfigured("procfs_path") {
+		procfsPath = pkgconfigsetup.Datadog().GetString("procfs_path")
+	}
 	return &NetworkCheck{
-		net:       defaultNetworkStats{},
+		net:       defaultNetworkStats{procPath: procfsPath},
```

Comment:
I believe we are not parsing the custom tags provided by customers via the `tags` configuration. 

In the Python check, we fetch them as part of the instance configuration 

```
custom_tags = self.instance.get('tags', [])
```

https://github.com/DataDog/integrations-core/blob/3fd29a6cf3ea69b09b8deec35bbce0e25b9578d3/network/datadog_checks/network/check_linux.py#L36

And ensure we pass it down to all the metrics. I believe we have to do that as well to ensure there are no missing tags

Reply:
I believe we do this in the BaseCheck when we call [CommonConfigure](https://github.com/DataDog/datadog-agent/blob/56a2bcf5836be9394851a2d379311291dcec8403/pkg/collector/corechecks/net/network/network.go#L703), it [will SetCheckCustomTags](https://github.com/DataDog/datadog-agent/blob/main/pkg/collector/corechecks/checkbase.go#L131) that are set on [the sender](https://github.com/DataDog/datadog-agent/blob/main/pkg/aggregator/sender.go#L129) included on every [sendMetricSample](https://github.com/DataDog/datadog-agent/blob/main/pkg/aggregator/sender.go#L186) which is used by [every type of send func](https://github.com/DataDog/datadog-agent/blob/main/pkg/aggregator/sender.go#L221-L263)

Reply:
Wonderful 😄